# This msg file defines an arm trajectory
# with (1) a list of JointState (2) armType

sensor_msgs/JointState[] trajectory
string armType